thunder/tests/opinfos.py

```diff
@@ -1278,16 +1278,10 @@ def _abs_torch(x: torch.Tensor | Number):
 elementwise_unary_ops.append(signbit_opinfo)
 
 
-def silu_error_generator(op, device, dtype=torch.float32, **kwargs):
-    a = make_tensor((), dtype=dtype, device=device)
-    yield (SampleInput(a, inplace=True), NotImplementedError, "Thunder only supports silu with inplace=False")
```

Comment:
Nice, so we record silu with inplace=True. In the subsymbols we use `prims.copy_` and then functionalization pass effectively changes `silu(x, inplace=True)` to just `silu(x)`.